docker/CMSRucioClient/scripts/syncRolesToAttributesLocalUsers.py

```diff
@@ -122,6 +160,12 @@ def sync_roles_to_rses():
                     local_users[site].add(dn)
                     dn_account_map[dn] = username
 
+        if 'group-data-manager' in roles:
```

Comment:
Make all of these setting things from roles look as similar as possible. We're now using different variables for the same thing and it will confuse the reader.